Pod/Classes/ATTutorialController.m

```diff
@@ -106,11 +116,11 @@ -(UILabel *)labelForTutorialWithText:(NSString *)title{
 
 -(FBShimmeringView *)labelForTutorialSwipe{
     CGRect frame = (CGRect){
-        {20.0, ((self.window.frame.size.height) - (60.0 + 20.0))}, // 60 = height of the label. 20 is the margin to the bottom.
-        {280.0, 60.0}
+        {0, ((self.window.frame.size.height) - (60.0 + 15.0))}, // 60 = height of the label. 20 is the margin to the bottom.
```

Comment:
These should also be for another PR. I think I have one hanging with autolayout and right positioning.